docs/application/dotnet/get-started/nui/quickstart.md

```diff
@@ -0,0 +1,286 @@
+    }
+    ```
+
+    >  **Note**  
+    > To invoke the `Terminated` event of the `Tizen.Applications.CoreApplication` class, you must call the `base.OnTerminate()` method inside the override.
+
+9.  Implement the `Main()` method:
+    1.  Create the application through the default constructor:
+
+        ```
+        static void Main(string[] args)
+        {   
+            var app = new Program();
+            app.Run(args);
+        }
+        ```
+
+    2.  Start the application main loop.
+
+        This ensures that images are displayed, and that events and signals are dispatched and captured.
+
+        In this simple tutorial, the `Main()` method resides within the class. For significant application development, the `Main()` method must be placed in a separate `.cs` file.
+
+        ```
+        app.Run(args);
+        ```
+
+## Building and Running NUI Application
+1.  Building the solution:
+
+    - In the Visual Studio menu, select **Build** > **Build Solution**.
+
+    - In the Solution Explorer view, right-click the solution name and select **Build**.
+
+2.  Launching Tizen Emulator:
+
+    - In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
+
+    - Alternatively, click **Launch Tizen Emulator** in the Visual Studio toolbar to launch the Tizen Emulator Manager:
+
+      ![launch_emulator1](media/tizen_project_wizard_capture_template3.png)
+
+    - In the Emulator Manager, select an emulator from the list, and click **Launch**:
+
+      ![launch_emulator2](media/tizen_project_wizard_capture_template4.png)
+
+    - You must update the following emulator images to use the latest version.
+
+3.  After you launch an emulator instance, you can select the target from the drop-down list to change the deployment target in the Visual Studio toolbar:
+
+    ![VS_Toolbar2](media/tizen_project_wizard_capture_template6.png)
+
+    You can deploy your NUI app to the target emulator in one of the following ways:
+
+    1. Press the **F5** key or **Ctrl+F5** key.
+
+    2. In the Visual Studio menu, select **Debug > Start Debugging** or **Start Without Debugging**.
+
+    3. On the Visual Studio toolbar, click an emulator instance:
+
+       ![VS_Toolbar](media/tizen_project_wizard_capture_template5.png)
+
+4.  If the deployment is successful, then the application is launched on the target emulator.
+
+    The following are snapshots of launching this NUI app on mobile emulators:
+
+    **Figure: NUI app on Mobile emulator**
+
+    ![NUIAppOnMobile](./media/nui-hello-world.png)
+
+    Visual Studio uses the Smart Development Bridge (SDB) to communicate with the target device or emulator.
+
+    You can also deploy your NUI application with SDB manually:
+
+    1.  Copy the application .tpk file from the project binary path to the SDB tool path:
+
+        - The `SampleNUIApp` tpk file(`org.tizen.example.SampleNUIApp-1.0.0.tpk`) is located in `ProjectPath\SampleNUIApp\SampleNUIApp\bin\Debug\tizen50`.
```

Comment:
Question. As the project path would be various for each user, isn't \<ProjectPath> correct?